src/test/ui/lint/invalid_value.stderr

```diff
@@ -34,6 +34,7 @@ LL |         let _val: Wrap<&'static T> = mem::zeroed();
    |                                      this code causes undefined behavior when executed
    |                                      help: use `MaybeUninit<T>` instead, and only call `assume_init` after initialization is done
    |
+   = note: `Wrap<&T>` must be non-null
 note: references must be non-null (in this struct field)
```

Comment:
This seems kind of redundant? `Wrap<&T>` must be non-null *because* references must be non-null (in this struct field).

But, fine with me either way. :shrug:

Reply:
I could add `because` to all messages where that makes sense

Reply:
Maybe that works? Not sure if we always have enough information around for that.